src/main/java/com/gmail/iikaliada/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.gmail.iikaliada.controller;
+
+import com.gmail.iikaliada.entity.User;
+import com.gmail.iikaliada.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
```

Comment:
It's better to add `"/users" ` here as it is a common path for all endpoints inside current controller